package.json

```diff
@@ -16,5 +16,6 @@
   },
   "devDependencies": {
     "lerna": "3.3.2"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
curious: has your team discussed using semantic versioning for this project at some point in the future?

Reply:
No idea, this should be 0.3.0 - fixing.